recipe/meta.yaml

```diff
@@ -28,7 +27,7 @@ requirements:
     - make          # [unix]
     # Require `cmake-no-system` to break circular dependency;
     # the `cmake` package on defaults requires `zstd`.
-    - cmake-no-system
+    - cmake-no-system 3.18.5
```

Comment:
The correct version is `3.16.5`

Reply:
this is the new dependency I am working on now, but it fails on s390, see https://github.com/AnacondaRecipes/cmake-no-system-feedstock/pull/5